test/unit/components/shepherd-element.spec.js

```diff
@@ -44,6 +44,44 @@ describe('components/ShepherdElement', () => {
         container.querySelectorAll('.shepherd-element .shepherd-arrow').length
       ).toBe(0);
     });
+
+    it('arrow: object with padding shows arrow', async () => {
+      const testElement = document.createElement('div');
+      const tour = new Tour();
+      const step = new Step(tour, {
+        arrow: { padding: 10 },
```

Comment:
Thanks for adding tests! Could we perhaps add an assertion checking that padding of `10px` is applied to the arrow element?